static/main.css

```diff
@@ -107,3 +107,39 @@ pre.sample-highlight { display:inline-block; }
 .slider-container {
     margin-bottom: 1em;
 }
+
+
+/* Only needed in  pair_view_summary.html */
+@media print {
+  .ex {
+    display: block !important;
+  }
+  .code-comparison { clear:both; overflow:visible !important; }
```

Comment:
Should I apply this to the whole main.css? Some old rules (e.g., histograms, comparison table) are compressed into single lines.

Reply:
Either way, changing old code is not mandatory. If you fix indentation in large parts of the old code without doing other changes, it would be best to make a separate git commit for that. It will be easier to follow the history when indentation changes are separated.